frontend/ui/widget/doublespinwidget.lua

```diff
@@ -167,17 +198,26 @@ function DoubleSpinWidget:update(numberpicker_left_value, numberpicker_right_val
 
     local buttons = {}
     if self.default_values then
+        local separator = self.is_range and "–" or "/"
+        local unit = ""
+        if self.unit then
+            if self.unit == "°" then
+                unit = self.unit
+            elseif self.unit ~= "" then
+                unit = "\xE2\x80\xAF" .. self.unit -- use Narrow No-Break Space (NNBSP) here
+            end
+        end
```

Comment:
Is it worth to add to utils `getUnitString`? Have 3 times of this code.

Reply:
I would not add this to `util`, as it is really very special.